content/docs/guides/polyscale.md

```diff
@@ -1,52 +1,90 @@
-![Create a PolyScale cache](/docs/guides/polyscale_create_cache.png)
-4. Click **Create**. PolyScale creates a cache for your Neon database and provides a new **Connection URI** to use in your application in place of the Neon connection string. Simply replace `[USERNAME]`, `[PASSWORD]`, and `[DATABASE]` with the values from your Neon connection string.
-![Create a PolyScale cache](/docs/guides/polyscale_success.png)
+    ![PolyScale manage drawer](/docs/guides/polyscale_int_manage.png)
 
-After queries from your application start passing through PolyScale, you can monitor traffic and caching behavior on the **Observability** tab in PolyScale. For more information, see [Observability](https://docs.polyscale.ai/database-observability/), in the _PolyScale documentation_.
+    Here, you can view information about your newly created PolyScale cache and copy the PolyScale connection string. Use this connection string in place of your Neon connection string in your application.
 
-By default, PolyScale automatically caches all queries that pass through the platform. That means you can connect to PolyScale, and any queries you run will be cached.
+    You might notice the drop-down menus for **Database** and **Role**. If you have multiple databases and roles on the primary branch of your Neon project, use the drop-down menus to select the database and role you want to connect with. Your selections will be reflected in the PolyScale connection string.
+   
+    <Admonition type="note">
+    To ensure your cache remains current, always use the PolyScale connection string. Modifying your database directly through a Neon connection string or the Neon SQL Editor can result in an outdated cache.
+    </Admonition>
 
-PolyScale identifies caching opportunities by recognizing and remembering patterns in query traffic. New queries typically begin to see cache hits on or about the third query. For more information, see [Time To First Hit](https://docs.polyscale.ai/how-does-it-work/#time-to-first-hit-ttfh), in the _PolyScale documentation_.
+## View your PolyScale cache
 
-For more information about using PolyScale and how it works, refer to the [PolyScale documentation](https://docs.polyscale.ai/).
+You can access your PolyScale cache directly from the Neon PolyScale integration.
 
-<NeedHelp/>
+1. In the Neon Console, navigate to the **Integrations** page.
+2. Under **Manage**, locate the PolyScale integration and click **Manage** to open the **Manage PolyScale** drawer.
+3. Click the **View cache in PolyScale** link. You directed to PolyScale where you will be required to log in if not logged in already.
+
+## Purging your PolyScale cache
+
+Occasionally, it may be necessary to purge your PolyScale cache. For example, you may want to purge your cache after modifying your data directly without using the PolyScale connection string, which can result in an outdated cache.
```

Comment:
```suggestion
Occasionally, it may be necessary to purge your PolyScale cache. For example, if you modify your data directly, without using the PolyScale connection string, this can result in an outdated cache. In this case, you may want to purge to keep the cache up to date.
```